src/components/PageSections/ContextualNav/index.js

```diff
@@ -22,7 +23,7 @@ const ContextualNav = ({ parent, relatedTo, offeredBy, intl }) => (
               {`${intl.formatMessage(i18n.relatedTo)}: `}
             </span>
             {relatedTo.map((relatedLinkData, index) => (
-              <Link to={`/${intl.locale}${relatedLinkData.url}`}>
+              <Link to={`/${intl.locale}${relatedLinkData.url}`} key={relatedLinkData.url}>
```

Comment:
The `index` alone would also work as a key, which is a common way people handle this in react. But the unique url also works.

Reply:
index as key is technically an antipattern, but since these components aren't going to be reordered, you're right that the key would have been just fine